src/e2e/java/uk/gov/hmcts/reform/sscs/functional/AbstractFunctionalTest.java

```diff
@@ -211,7 +212,12 @@ private String generateScNumber(Instant currentEpoh, Map<String,String> sscCodeM
     }
 
     protected void simulateCcdCallback(NotificationEventType eventType) throws IOException {
-        String resource = eventType.getId() + "Callback.json";
+        String resource = "";
+        if (APPEAL_RECEIVED_NOTIFICATION.equals(eventType)) {
```

Comment:
perhaps call `simulateCcdCallback(NotificationEventType eventType, String resource)` instead for `APPEAL_RECEIVED_NOTIFICATION` rather than use this method.